### ISSUE TITLE
Non-nullable extra column with default value fails with --overwrite

### ISSUE THREAD

**Issue by Nikheln** [ankane/pgsync#113]
Given a source database with the source table defined and populated with
```sql
create table public.test (
    id int primary key,
    value_1 boolean
);

insert into public.test values (1);
```
and a target database with a table defined as
```sql
create table public.test (
    id int primary key,
    value_1 boolean,
    value_2 boolean not null default false
);
```
the synchronization works as expected when executed without `--overwrite`. New rows synchronized to the target get the defined default value `false` in column `value_2`.

However, when running the same synchronization with the `--overwrite` flag, the following error is raised from the target database:
```
ERROR:  null value in column "value_2" violates not-null constraint
DETAIL:  Failing row contains (1, null, null).
STATEMENT:  INSERT INTO "public"."test" (SELECT * FROM "pgsync_175830146") ON CONFLICT ("id") DO UPDATE SET "value_1" = EXCLUDED."value_1"
```

**Version info:**
pgsync: `0.6.5`
PostgreSQL: `PostgreSQL 12.2 (Debian 12.2-2.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit`

**Comment by ankane** [comment]
Hey @Nikheln, that should be fixed in 0.6.6.

**Comment by Nikheln** [comment]
So it seems, I rebuilt my environment with 0.6.6 and the issue disappeared, thank you!
